docs/Configuration.md

```diff
@@ -4,18 +4,97 @@ title: Configuration
 permalink: /configuration/
 nav_order: 4
 ---
-# Configuration 
+
+# Configuration
 
 VS Code settings enable you to customize various aspects of its function. The InterSystems extensions provide settings used to configure VS Code for ObjectScript development.
 
+{: #code-configuration-basic}
+
+## Basic Configuration
+
+VS Code has a concept of a workspace, which is a set of directories you want to use when you're working on a particular project. In the simplest setup when you are working within a single directory, a VS Code workspace is just the root folder of your project. In this case you keep your settings in two files inside a `.vscode` directory located at the root of your project. Those two files are `settings.json`, which contains most configuration settings, and `launch.json`, which contains debugging configurations.
+
+Here is the simplest `settings.json` file contents:
+
+{: #code-workspace-simple}
+
+```json
+{
+    "objectscript.conn": {
+        "server": "iris",
+        "ns": "USER",
+        "active": true, 
+        "host": "localhost", 
+        "port": 52773, 
+        "username": "_SYSTEM" 
+    }
+}
+```
+
+If you need ObjectScript compilation flags other than the default, add an `"objectscript.compileFlags"` property to `settings.json` (more compileFlags options are [covered here](/vscode-objectscript/settings/):
```

Comment:
I don't think this setting needs to be covered here. It has a detailed description in the `package.json` file and Settings Reference page and users who know enough about our technology to want to change it will be able to figure out how to input their flags/qualifiers.

Reply:
I found it unintuitive to mix single-letter flags with longer qualifiers, and we had an internal user not find the solution also. I think some duplication is good when it comes to documentation.

Reply:
The internal user was unaware of a behavior change w.r.t. when embedded SQL is compiled that was introduced in 2020.1. Having this paragraph added wouldn't address their problem because they didn't know that they needed the `/compileembedded` qualifier, not that they didn't know where to specify it.

Reply:
Ah got it @isc-bsaviano. I still feel that the syntax is awkward though. After I identified the `/compileembedded` qualifier in the docs, it felt very strange to write `"cuk/compileembedded=1"` instead of, e.g., `c,u,k,/compileembedded=1`, so I think this example can potentially help but never hurt.